lib/drivers/onetouch/oneTouchUltra2.js

```diff
@@ -213,43 +199,73 @@ module.exports = function (config) {
      }
 
     // get data from lines
-    var dpattern = /P "(\w+)","(\d{2}\/\d{2}\/\d{2})","(\d{2}:\d{2}:\d{2})   ","..(\d{3}).","(.)","(\d{2})",\W00\W(.+)/;
+    var dpattern = /P "(\w+)","(\d{2}\/\d{2}\/\d{2})","(\d{2}:\d{2}:\d{2})   ","(..\d{3}).","(.)","(\d{2})",\W00\W(.+)/;
```

Comment:
The regex for the glucose value does not capture the optional trailing question mark which signals a parity error according to the specs. Is that intentional, or should there be some specific handling of values with this flag?